src/crate/crash/command.py

```diff
@@ -235,9 +235,9 @@ def exit(self):
     @noargs_command
     def _show_tables(self, *args):
         """ print the existing tables within the 'doc' schema """
-        self._exec("""select format('%s.%s', schema_name, table_name) as name
+        self._exec("""select format('%s.%s', table_schema, table_name) as name
```

Comment:
this command should check for the server version and then execute the correct statement accordingly